provision/provisioners.sh

```diff
@@ -41,13 +45,13 @@ function provisioner_end() {
   trap - EXIT
 }
 
-if [[ ! -z $VVV_LOG ]]; then
+if [[ -n $VVV_LOG ]]; then
```

Comment:
this change is right?

Reply:
yeah it's based on shellcheck feedback https://www.shellcheck.net/wiki/SC2236